foundation/foundation.css

```diff
@@ -0,0 +1,133 @@
+body {
+    font-family: Arial,Helvetica,Verdana,sans-serif;
+    font-size: 15px;
+    line-height: 22px;
+    color: #666666;
+}
+
+h1 {
+    font-size: 32px;
+    line-height: 36px;
+    color: #333333;
+    padding-bottom: 18px;
+}
+
+h2 {
+    font-size: 24px;
+    line-height: 22px;
+    color: #333333;
+    padding-bottom: 14px;
+}
+
+h3 {
+    font-size: 18px;
+    line-height: 22px;
+    color: #333333;
+    padding-bottom: 6px;
+}
+
+p {
+	  color: ;
+	  font-size: 15px;
+    line-height: 22px;
+    color: #666666;
+    padding-bottom: 6px;
+}
+
+small {
+	  font-size: 13px;
+    line-height: 18px;
+    color: #666666;
+}
+
+.micro {
+    font-size: 11px;
+    line-height: 13px;
+    color: #666666;
+    text-transform: uppercase;
+}
+
+/* BACKGROUNDS COLORS - grey */
+.darkgrey {
+    background-color: #333333; /* text */
+}
+
+.grey {
+	  background-color: #666666; /* icons fonts */
```

Comment:
Please check alignment through all the css, maybe you have an issue with your code editor.

Reply:
watch out! is true alignments are not declared in the headings, but doesn't seem right to me declare alignments in color classes